pkg/kubectl/describe.go

```diff
@@ -2385,6 +2387,36 @@ func (s *StorageClassDescriber) Describe(namespace, name string, describerSettin
 	})
 }
 
+type PodDisruptionBudgetDescriber struct {
+	clientset.Interface
+}
+
+func (p *PodDisruptionBudgetDescriber) Describe(namespace, name string, describerSettings DescriberSettings) (string, error) {
```

Comment:
I guess we should print some status information as well. However I have a pr in flight that changes the status struct a bit #34546. The PR should be merged on Monday.